estate/models/estate_property.py

```diff
@@ -0,0 +1,24 @@
+from odoo import models, fields
+
+class EstatePropertyModel(models.Model):
+    _name = "estate.property"
+    _description = "Estate Propery Model"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date()
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float()
+    bedrooms = fields.Integer()
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
```

Comment:
```suggestion
    is_garage = fields.Boolean()
    is_garden = fields.Boolean()
```
Update this fields name to make it more effective to use in code (Try this type of names for boolean fields)